core/src/test/scala/org/apache/spark/SparkFunSuite.scala

```diff
@@ -299,10 +299,15 @@ abstract class SparkFunSuite
       parameters: Map[String, String] = Map.empty,
       matchPVals: Boolean = false,
       queryContext: Array[QueryContext] = Array.empty): Unit = {
-    assert(exception.getErrorClass === errorClass)
+    val mainErrorClass :: tail = errorClass.split("\\.").toList
+    assert(tail.isEmpty || tail.length == 1)
+    // TODO: remove the `errorSubClass` parameter.
```

Comment:
just nit. If we use IDed TODO, some contributor can pick up the item more easily.

Reply:
I didn't create a JIRA for this TODO because @MaxGekk will fix it shortly (we talked offline) :)